Description: Decrease alsa buffer size. That improves the behaviour of the alsa
 output module on pulseaudio system. But note that vlc-plugin-pulse provides a
 native pulseaudio output module.
Author: Reinhard Tartler <siretart@tauware.de>
Bug-Debian: http://bugs.debian.org/472811
Bug-Ubuntu: https://launchpad.net/bugs/243152

Index: vlc-debian/modules/audio_output/alsa.c
===================================================================
--- vlc-debian.orig/modules/audio_output/alsa.c	2010-02-13 15:44:11.000000000 +0000
+++ vlc-debian/modules/audio_output/alsa.c	2010-02-13 15:44:27.000000000 +0000
@@ -78,7 +78,7 @@
    number of channel(s) (eg. 2 for stereo) and the size of a sample (eg.
    2 for int16_t). */
 #define ALSA_DEFAULT_PERIOD_SIZE        1024
-#define ALSA_DEFAULT_BUFFER_SIZE        ( ALSA_DEFAULT_PERIOD_SIZE << 8 )
+#define ALSA_DEFAULT_BUFFER_SIZE        ( ALSA_DEFAULT_PERIOD_SIZE << 4 )
 #define ALSA_SPDIF_PERIOD_SIZE          A52_FRAME_NB
 #define ALSA_SPDIF_BUFFER_SIZE          ( ALSA_SPDIF_PERIOD_SIZE << 4 )
 /* Why << 4 ? --Meuuh */
